concourse/pipelines/create-deployer.yml

```diff
@@ -467,6 +467,10 @@ jobs:
                               ./paas-cf/manifests/concourse-manifest/extensions/datadog-agent.yml
                               ./paas-cf/manifests/shared/deployments/datadog-agent.yml"
             fi
+            # shellcheck disable=SC2050
```

Comment:
In case of doubt these are the changes I propose:
```diff
diff --git a/Makefile b/Makefile
index 8b4734d..0f422ce 100644
--- a/Makefile
+++ b/Makefile
@@ -146,6 +146,7 @@ prod: globals check-env-vars ## Set Environment to Production
 	$(eval export ENABLE_CF_ACCEPTANCE_TESTS=false)
 	$(eval export ENABLE_DATADOG=true)
 	$(eval export DECRYPT_CONCOURSE_ATC_PASSWORD=prod_deployment)
+	$(eval export CONCOURSE_AUTH_DURATION=5m)
 	@true
 
 .PHONY: bootstrap
diff --git a/concourse/pipelines/create-deployer.yml b/concourse/pipelines/create-deployer.yml
index c4ff5eb..7ea357c 100644
--- a/concourse/pipelines/create-deployer.yml
+++ b/concourse/pipelines/create-deployer.yml
@@ -442,6 +442,7 @@ jobs:
           AWS_ACCOUNT: {{aws_account}}
           DATADOG_API_KEY: {{datadog_api_key}}
           ENABLE_DATADOG: {{enable_datadog}}
+          CONCOURSE_AUTH_DURATION: {{concourse_auth_duration}}
           CONCOURSE_MANIFEST_STUBS: |
             ./paas-cf/manifests/concourse-manifest/concourse-base.yml
             predefined-concourse-secrets/predefined-concourse-secrets.yml
@@ -467,10 +468,6 @@ jobs:
                               ./paas-cf/manifests/concourse-manifest/extensions/datadog-agent.yml
                               ./paas-cf/manifests/shared/deployments/datadog-agent.yml"
             fi
-            # shellcheck disable=SC2050
-            if [ {{aws_account}} = "prod" ] ; then
-              export AUTH_DURATION=5m
-            fi
             ./paas-cf/manifests/shared/build_manifest.sh $CONCOURSE_MANIFEST_STUBS \
               > concourse-manifest/concourse-manifest.yml
             ls -l concourse-manifest/concourse-manifest.yml
diff --git a/concourse/scripts/pipelines-deployer.sh b/concourse/scripts/pipelines-deployer.sh
index 83c8de9..f22f788 100755
--- a/concourse/scripts/pipelines-deployer.sh
+++ b/concourse/scripts/pipelines-deployer.sh
@@ -32,6 +32,7 @@ system_dns_zone_name: ${SYSTEM_DNS_ZONE_NAME}
 aws_account: ${AWS_ACCOUNT:-dev}
 datadog_api_key: ${datadog_api_key:-}
 enable_datadog: ${ENABLE_DATADOG}
+concourse_auth_duration: ${CONCOURSE_AUTH_DURATION:-24h}
 EOF
 }
 
diff --git a/manifests/concourse-manifest/concourse-base.yml b/manifests/concourse-manifest/concourse-base.yml
index fa07316..9bbc001 100644
--- a/manifests/concourse-manifest/concourse-base.yml
+++ b/manifests/concourse-manifest/concourse-base.yml
@@ -81,7 +81,7 @@ jobs:
           external_url: (( concat "https://" terraform_outputs.concourse_dns_name ))
           basic_auth_username: admin
           basic_auth_password: (( grab secrets.concourse_atc_password ))
-          auth_duration: (( grab $AUTH_DURATION || "24h" ))
+          auth_duration: (( grab $CONCOURSE_AUTH_DURATION || null ))
 
           postgresql:
             address: 127.0.0.1:5432
```